augur/tasks/github/util/github_random_key_auth.py

```diff
@@ -16,6 +17,7 @@ def __init__(self, session: DatabaseSession, logger):
     
         # gets the github api keys from the database via the GithubApiKeyHandler
         github_api_keys = GithubApiKeyHandler(session).keys
+        github_api_keys = random.sample(github_api_keys, len(github_api_keys))
 
```

Comment:
I believe the goal here is to randomize the order. If so random.shuffle should be used since Random.sample is used to get a random subset of a list. This will randomly order the list but it is less clear what the goal is